### PR TITLE
init types for memorystream

### PR DESCRIPTION
**Pull request by bangbang93**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/memorystream/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+// Type definitions for memorystream 0.3
+// Project: https://github.com/JSBizon/node-memorystream
+// Definitions by: My Self <https://github.com/me>
```
Comment (DanielRosenwasser) [inline types/memorystream/index.d.ts]: Please add yourself as an author.

File: types/memorystream/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    readable: boolean;
+    writeable: boolean;
+    maxbufsize: number;
+    bufoverflow: number;
+    frequence: number;
+}
+
+declare class MemoryStream extends Duplex {
+    constructor(
+        data?: DataType | DataType[],
```
Comment (DanielRosenwasser) [inline types/memorystream/index.d.ts]: Is this actually potentially an array of strings?

Comment (bangbang93) [reply types/memorystream/index.d.ts]: yes it is
image

**Comment by DanielRosenwasser** [comment]
Thanks!

**Pull request merged**